test/state/TestStateNode.java

```diff
@@ -182,8 +182,8 @@ public void testIdentifier() {
     private Sig createNewSig() {
         Sig sigA = new Sig.PrimSig("A");
         Sig sigB = new Sig.PrimSig("B");
-        Sig.Field f1 = sigA.addField("f", sigB.lone_arrow_lone(sigB));
-        Sig.Field f2 = sigA.addField("g", sigB);
+        Sig.Field f1 = sigA.addField("g", sigB);
+        Sig.Field f2 = sigA.addField("f", sigB.lone_arrow_lone(sigB));
```

Comment:
switched the field names around to ensure that the TreeMap correctly sorted in lexicographical order.